lib/bundler/cli/gem.rb

```diff
@@ -29,6 +29,11 @@ def run
       constant_name = name.gsub(/-[_-]*(?![_-]|$)/) { "::" }.gsub(/([_-]+|(::)|^)(.|$)/) { $2.to_s + $3.upcase }
       constant_array = constant_name.split("::")
 
+      if !Bundler.git_present?
+        Bundler.ui.error "You need to install git to be able to use bundle gem."
```

Comment:
@colby-swandale if that's the case, then this is not the right fix. I see your point in allowing it but it's not clear to me from bundler doc's if `bundle gem` is allowed to work without `git` or it requires `git`. Let's clarify this so the correct fix is implemented.

@indirect @segiddins any thoughts on git requirement for `bundle gem`?